{
  "id": "CVE-2025-48392",
  "sourceIdentifier": "security@apache.org",
  "published": "2025-09-24T08:15:31.880",
  "lastModified": "2025-09-24T18:11:24.520",
  "vulnStatus": "Awaiting Analysis",
  "descriptions": [
    {
      "lang": "en",
      "value": "A vulnerability in Apache IoTDB.\n\nThis issue affects Apache IoTDB: from 1.3.3 through 1.3.4, from 2.0.1-beta through 2.0.4.\n\nUsers are recommended to upgrade to version 2.0.5, which fixes the issue."
    }
  ],
  "metrics": {},
  "references": [
    {
      "url": "https://lists.apache.org/thread/1rn0637hptglmctf8cqd9425bj4q21td",
      "source": "security@apache.org"
    }
  ]
}